Allow setting RAXSDK_TIMEOUT, but define default timeout if not previously defined

carlaron opened the pull request:
I needed to over-ride the 5 minute time-out (we have people uploading large videos)

This allows me to place the define() command in my own code, and have it take precedence over the definition in OpenCloud, which is still needed by anyone who hasn't otherwise defined it.

If you can think of a better way, I'm open to suggestions

Thanks!

thomaswelton commented:
I don't really use this plugin for my own purposes anymore so it's not really maintained.
If you can update this so that the RAXSDK_TIMEOUT value can be set in a Laravel config file then I'll merge it. If not you'll have to use your own fork.
Cheers.

carlaron commented:
Thanks! That's a good idea. I added a key raxsdk_timeout to the existing config file, and changed OpenCloud.php to use that when setting the RAXSDK_TIMEOUT constant.

thomaswelton commented:
Published as version 0.4.1

carlaron commented:
Thanks for your fast response!
